src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidatorBuilder.cs

```diff
@@ -82,6 +125,30 @@ public IMarkdownTokenRewriter Create()
                 });
         }
 
+        private IEnumerable<MarkdownTagValidationRule> GetEnabledTagRules()
+        {
+            foreach (var item in _tagValidators)
+            {
+                if (item.FullId != null)
+                {
+                    MarkdownValidationRule rule;
+                    if (_validators.TryGetValue(item.FullId, out rule) ||
+                        _validators.TryGetValue(item.Category, out rule))
+                    {
+                        if (!rule.Disable)
+                        {
+                            yield return item.TagRule;
```

Comment:
fullid.Disable > category.Disable > tagRule.Disable

---
In reply to: [72755987](https://github.com/dotnet/docfx/pull/545#discussion_r72755987) [](ancestors = 72755987)